framework/utils/CPropertyValue.php

```diff
@@ -113,8 +113,14 @@ public static function ensureArray($value)
 			$len = strlen($value);
 			if ($len >= 2 && $value[0] == '(' && $value[$len-1] == ')')
 			{
-				eval('$array=array'.$value.';');
-				return $array;
+				try
+				{
+					return @eval('return array' . $value . ';');
```

Comment:
For suppressing errors from eval() function. I think in this case we will simply parse a necessary exception or get clear false without any additional messages. Similar solution you can see \CChoiceFormat::evaluate or in the \CAuthManager::executeBizRule or \CTimestampBehavior::getTimestampByAttribute